dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -257,19 +281,11 @@ private Set<TrackedEntityAttributeValue> getTrackedEntityAttributeValues(Enrollm
   }
 
   private List<Enrollment> findEnrollments(
-      Iterable<Enrollment> enrollments,
-      EnrollmentFields fields,
-      boolean includeDeleted,
-      OrganisationUnitSelectionMode orgUnitMode) {
+      Iterable<Enrollment> enrollments, EnrollmentFields fields, boolean includeDeleted) {
     List<Enrollment> enrollmentList = new ArrayList<>();
-    UserDetails currentUser = getCurrentUserDetails();
 
     for (Enrollment enrollment : enrollments) {
-      if (enrollment != null
-          && (orgUnitMode == ALL
-              || trackerOwnershipManager.hasAccess(
-                  currentUser, enrollment.getTrackedEntity(), enrollment.getProgram()))
-          && trackerAccessManager.canRead(currentUser, enrollment, orgUnitMode == ALL).isEmpty()) {
+      if (enrollment != null) {
```

Comment:
How can the enrollment be `null` here?
If it is possible, then we are still breaking pagination.

Reply:
That's a remnant of the old ACL validation and is no longer needed.